### PR TITLE
Fix up the model for release

### PR DESCRIPTION
**Pull request by davidfischer**
- Remove game-dev, techwriting, and hardware topics (for now)
- Remove navigation from the analyzer
- Add additional examples that weren't classified well

### CODE REVIEW

**Comment by davidfischer** [comment]
```
================================== Results ==================================

TOK                   100.00
TEXTCAT (macro AUC)   90.32 
SPEED                 28336 


=========================== Textcat F (per label) ===========================

                   P       R       F
datascience    90.91   90.91   90.91
backend        75.00   18.75   30.00
frontend       50.00   11.11   18.18
security       66.67   18.18   28.57
devops         81.25   56.52   66.67
blockchain    100.00   80.00   88.89


======================== Textcat ROC AUC (per label) ========================

              ROC AUC
datascience      0.96
backend          0.79
frontend         0.87
security         0.91
devops           0.89
blockchain       0.99
```
Still more to go but getting better.

File: adserver/analyzer/backends/naive.py [hunk 1]
```diff
@@ -23,6 +23,18 @@ class NaiveKeywordAnalyzerBackend(BaseAnalyzerBackend):
         "body",
     )
 
+    # CSS selectors of content that should be ignored for the purpose of analysis
```
Comment (ericholscher) [inline adserver/analyzer/backends/naive.py:26]: This is the same logic that was in the training import code, just ported to the backend, right?

File: machine_learning_experiments/scripts/preprocess.py [hunk 2]
```diff
@@ -13,10 +13,10 @@
     "frontend",
     "security",
     "devops",
-    "game-dev",
+    # "game-dev",
```
Comment (ericholscher) [inline machine_learning_experiments/scripts/preprocess.py:16]: I'm a little confused on our semantics around slugging here. I would expect `data-science` and `tech-writing` to be hyphenated if we're doing that, but it seems we're not consistent. Should we define how we're slugging things, so we can try to be consistent, as well as asking our users to be consistent.

**Review by ericholscher — APPROVED** [review]
Solid improvements. I'm a little worried about all the content we're tagging as "no topic" and how that might impact the model. I feel like there's tons of python/node stuff in the documentation tooling that will probably be focusing for it when it's also very present in backend tags for example. But perhaps that's how we make it more discerning. 

> frontend       50.00   11.11   18.18

I'm surprised frontend is so low. That definitely feels like the next area to improve. Perhaps because we don't have as much content-heavy sites in that area?

File: adserver/analyzer/backends/naive.py (hunk 1 above)
Comment (davidfischer) [reply adserver/analyzer/backends/naive.py:26]: Correct, ported to the analyzer.

File: machine_learning_experiments/scripts/preprocess.py (hunk 2 above)
Comment (davidfischer) [reply machine_learning_experiments/scripts/preprocess.py:16]: Sure. The important thing is that there has to be a keyword with a matching slug (we have a keyword "data science" and "datascience" so either way works).

**Comment by davidfischer** [comment]
> I'm surprised frontend is so low. That definitely feels like the next area to improve. Perhaps because we don't have as much content-heavy sites in that area?

This is definitely an area for improvement.

**Pull request merged**